### ISSUE TITLE
[MAS2.1.1] Check-boxes present in 'Emulator Settings' pane are not accessible using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#740]
**Actual Result:**
Check-boxes present in Emulator Settings pane are not accessible using keyboard.

**Expected Result:**
Check-boxes present in Emulator Settings pane should be accessible using keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome screen will open up.
Navigate to Settings button present in bottom of Left pane and select it.
3. In 'Emulator Settings' pane, navigate using Tab key and notice that focus does not come at check-boxes present in this pane.

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachments:**
[MAS2.1.1_BotFrameworkEmulator_EmulatorSettings_CheckBoxes.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2266368/MAS2.1.1_BotFrameworkEmulator_EmulatorSettings_CheckBoxes.zip)

**Comment by corinagum** [comment]
Unable to repro on latest version of v4. I am able to flip the values of the checkbox using both spacebar and enter. Visually, the entire input is surrounded with blue box on focus.

However, on **focus** and **hover** in dark and high contrast themes, the color of the font is dark and becomes illegible. I will fix that as a part of this bug.
